cmd/xl-v1-list-objects.go

```diff
@@ -91,7 +110,7 @@ func (xl xlObjects) listObjects(bucket, prefix, marker, delimiter string, maxKey
 			objInfo, err = xl.getObjectInfo(bucket, entry)
 			if err != nil {
 				// Ignore errFileNotFound
```

Comment:
Comment should be fixed.. perhaps we can now use `isErrIgnored()` ?

Reply:
ummm, errFileNotFound should be ignored too because the object might have got deleted in the interim period of listing and getObjectInfo()